src/test/eo/org/eolang/math/integral-tests.eo

```diff
@@ -0,0 +1,88 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2022 Max Trunnikov
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++alias org.eolang.hamcrest.assert-that
++alias org.eolang.math.integral
++alias org.eolang.math.number
++home https://github.com/objectionary/eo-math
++junit
++package org.eolang.math
++version 0.0.0
+
+[] > linear-integral-test
+  [x] > lin
+    x > @
+  assert-that > @
+    integral
+      lin
+      1
+      10
+    $.close-to
+      49.5
+      0.0000001
+
+[] > quadratic-integral-test
+  [x] > quadr
+    pow. > @
+      number x
+      2.0
+  assert-that > @
+    integral
+      quadr
+      1
+      10
+    $.close-to
+      333.0
+      0.0000001
+
+[] > cube-integral-test
+  [x] > cube
+    pow. > @
+      number x
+      3.0
+  assert-that > @
+    integral
+      cube
+      1
+      10
+    $.close-to
+      2499.75
+      0.0000001
+
+# @todo #93:30min This test fails due to
+#  incorrect choose of steps in the 
+#  integral implementation. For excluding
```

Comment:
@includealex I don't get it. Maybe you mean incorrect logarithm implementation?

Reply:
@Graur in the current implementation we take number of sections equal to `n = 100`. That is rude and some tests will fail, for instance, with logarithm. We should use this rule
![image](https://user-images.githubusercontent.com/70340829/204376056-edccb919-64a4-4e28-8cd5-d6f57ca5cdfe.png)
For counting it in more accurate way, and that depends on integral implementation.

Reply:
@includealex Understood. Thanks!